### PR TITLE
Allow customizable module versions and bump configured versions.

### PR DESCRIPTION
**Pull request by jsuereth**
Review/use by @dotta

### CODE REVIEW

File: sbt-on-2.11.x [hunk 1]
```diff
@@ -3,8 +3,12 @@
   globals: {
     scala-version: "2.11.0-SNAPSHOT"
     scala-version: ${?SCALA_VERSION}
-    scala-binary-version: "2.11.0-M4"
+    scala-binary-version: "2.11.0-M5"
     scala-binary-version: ${?SCALA_BINARY_VERSION}
+    parser-combinator-version: "1.0.0-RC3"
+    parser-combinator-version: ${?SCALA_PARSER_COMBINATOR_VERSION}
+    xml-version: "1.0.0-RC5"
+    xml-version: ${?SCALA_XML_VERSION}
```
Comment (dotta) [inline sbt-on-2.11.x:11]: Maybe silly question, but why do we have `xml-version` and `parser-combinator-version` twice?

Actually, everything is duplicated, so I'm guessing the first is a default value?!

**Comment by dotta** [comment]
This looks good to me!

File: sbt-on-2.11.x (hunk 1 above)
Comment (jsuereth) [inline sbt-on-2.11.x:11]: In your script, you can use the environment variable to override the default value.  A quirk of typesafe config library.

Comment (dotta) [inline sbt-on-2.11.x:11]: Thanks for the explanation!

**Comment by adriaanm** [comment]
LGTM - thanks!

**Pull request merged**